Sources/Sentry/SentryScreenshot.m

```diff
@@ -0,0 +1,31 @@
+#import "SentryScreenShot.h"
+#import "SentryUIApplication.h"
+
+#if SENTRY_HAS_UIKIT
+#    import <UIKit/UIKit.h>
+
+@implementation SentryScreenshot
+
+- (NSArray<NSData *> *)appScreenshots
```

Comment:
It is a little expensive.

Reply:
I don't think we should point this out on documentation. Developers should have a little notion on how expensive is to draw the entire screen.
And anyway, this captures are only done on-demand, is not like this would be harmful to the app execution. 

Writing these kind of comments can drive users away from the feature.

Reply:
> Writing these kind of comments can drive users away from the feature.

💯